src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
```

Comment:
4. Use an instance variable for the Random object and the ColorSupplier.
5. Now that we have a parameterized constructor, we don't need all that.
We'll just return new Ball(randomColor, randomNumber).
This values we can create as separate varibales to better readability.
String randomColor = colorSupplier.getRandomColor();